controllers/xmlrpc/src/main/java/orca/controllers/xmlrpc/OrcaXmlrpcHandler.java

```diff
@@ -1882,7 +1882,12 @@ protected static Properties getEthModifyProperties(String parent_prefix, String
 			// check the whitelist
 			if (verifyCredentials && !checkWhitelist(userDN)) 
 				return setError(WHITELIST_ERROR);
-			
+
+			if (!verifyCredentials && null == userDN){
```

Comment:
Yeah, I haven't been thrilled about that change either.  But I think when running live, the `verifyCredentials` is turned on (even in emulation).  

In my unit tests I have turned `verifyCredentials` off, but that gives me no way to set the userDN, unless I make that somewhat horrible hack.